tests/high-level-tests.lisp

```diff
@@ -38,5 +38,10 @@
   "Check that the kronecker product of two small identities is a larger identity."
   (let* ((matrix-dim 2)
          (eye2 (make-identity-matrix matrix-dim))
-         (eye4 (kron eye2 eye2)))
-    (is (magicl:identityp (magicl:make-identity-matrix (* matrix-dim matrix-dim))))))
+         (eye4 (kron eye2 eye2))
+         (x (magicl:make-complex-matrix matrix-dim matrix-dim '(0 1 1 0)))
+         (xxx (kron x x x)))
+    (is (magicl:identityp (magicl:make-identity-matrix (* matrix-dim matrix-dim))))
+    (is (loop :for i :below (matrix-rows xxx) :always
+                            (loop :for j :below (matrix-cols xxx)
```

Comment:
please extract `(matrix-cols xxx)` via `let`

Reply:
got it!